.github/workflows/publish-pypi.yml

```diff
@@ -53,32 +40,39 @@ jobs:
     needs: [build]
     environment:
       name: pypi
-      url: https://pypi.org/p/pipecat-cloud
+      url: https://pypi.org/p/pipecatcloud
     permissions:
       id-token: write
-      contents: write # Needed for tagging
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
+      - name: Download wheels
+        uses: actions/download-artifact@v4
         with:
-          fetch-tags: true
-          fetch-depth: 100
-
-      - name: Create release tag
-        run: |
-          git config --local user.email "action@github.com"
-          git config --local user.name "GitHub Action"
-          git tag -a "v${{ github.event.inputs.version }}" -m "Version ${{ github.event.inputs.version }}"
-          git push origin "v${{ github.event.inputs.version }}"
+          name: wheels
+          path: ./dist
+      - name: Publish to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          verbose: true
+          print-hash: true
 
+  publish-to-test-pypi:
```

Comment:
We have the workflow below publishing to Test PyPI. Isn’t this one supposed to publish to PyPI itself ?